backend/contentBackend.js

```diff
@@ -36,54 +36,52 @@ async function authUserAndValidateRoute(jwt, route, invalidRouteError) {
  * into its valid HTML repr. Also returns frontmatter
  * attributes extracted from the original content.
  *
- * @param { string } jwt - token used for identification
+ * @param { string }  - token used for identification
  * @param { string } markdownContent - content to parse
  *
  * @return { object } - parsed content and attributes
  */
 /* @expose */
-export async function parseMD(jwt, markdownContent) {
-  await validateJWT(jwt);
+export async function parseMD(markdownContent) {
+  await validateUser();
+
   return parseMDLocal(markdownContent);
 }
 
 /**
  * Set the site's home route
  *
- * @param { string } jwt - token used for identification
+ * @param { string }  - token used for identification
  * @param { string } route - route to set as home
  *
  */
 /* @expose */
-export async function setRouteAsHome(jwt, route) {
-  await authUserAndValidateRoute(jwt, route, 'Cannot set undefined or null route as home');
+export async function setRouteAsHome(route) {
+  await authUserAndValidateRoute(route, 'Cannot set undefined or null route as home');
```

Comment:
AFAICT this message is always the same, only the action is different (`set` here).  Why not pass _just_ the action?

Reply:
Yes it does seem pretty redundant, but there is small discrepancies in the error messages making them `a little` different would you recommend making them all uniform or keep as is?